wasm/wasi/wit/deps/hermes-logging/api.wit

```diff
@@ -32,12 +32,12 @@ interface api {
     /// The Webasm module does not need to concern itself with this kind of information, and should
     /// log as if it is the only instance.
     /// It also should not log any webasm shared context, except where it is relevant to the log message itself.
-    ///
+    /// The log level will be forced to INFO level.
+    /// 
     /// **Parameters**
     ///
-    /// - `level` : The log level this message is for. 
```

Comment:
You must have level, we need to know what level the wasm module assigned to the log message